introduction.md

```diff
@@ -1,2 +1,6 @@
 My name is Moises Arcos and I'm Software Engineer. I'm working as a GIS developer at Emergya in Seville and I participate in a local investigation group call it "Geoinquietos". The aim of this group is divulgate GIS knowledge and promote event for the citizens.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?